fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/model/DatabaseObject.java

```diff
@@ -195,7 +195,7 @@ public void applyTx(IDatabaseAdapter target, ITransactionProvider tp, IVersionHi
     public void applyVersion(IDatabaseAdapter target, IVersionHistoryService vhs) {
```

Comment:
Have you tried deploying the 4.1.0 schema jar, and upgrade. from there?  I'm curious if we have any latent issues.

Reply:
I updated Lee's unit test which performs the migration.

Reply:
Yes, it's also covered in the CI, which has passed from 4.2.3 to this